feat(dedibox-domains): removed domain name ownership change MTA-7391 (#6809)
scaleway/docs-content: pages/dedibox-domains/how-to/purchase-domain.mdx

    <Message type="tip">
      If the domain name is not available because you are already the owner, you have the option to transfer the domain so that Scaleway becomes your new registrar, and you can manage your hosting and domain from the same location. See the [relevant documentation](/dedibox-domains/how-to/transfer-domain/) for more information.
    </Message>
3. Select an available domain-name/extension combination from the list by clicking the checkbox next to it.
    <Message type="note">
      Some Top Level Domain (TLD) extensions have special requirements that owners must comply with. You may be asked for verification if you order or renew one of these extensions. [Learn more about TLD specifications](/dedibox-domains/concepts/#tld-specifications-at-scaleway).
    </Message>
4. Scroll to the bottom of the page and click **Order**. The **Hosting** page opens.
5. Select a web hosting package, or choose to proceed with no hosting. You can also add a hosting backup if you wish.
    <Message type="note">
      If you purchase web hosting, you get your domain name for free with purchase/renewal.
    </Message>
6. Scroll down to the bottom of the page and click **Order**.

> 7. If you did not select hosting backup, a pop-up may display asking you to confirm your choice. Click **Continue without backup** or **Add backup**. The **Contact information** page opens.

Reviewer: ```suggestion
7. If you did not select hosting backup, a confirmation dialog box may appear. Click **Continue without backup** or **Add backup**. The **Contact information** page opens.
```

> 7. If you did not select hosting backup, a pop-up may display asking you to confirm your choice. Click **Continue without backup** or **Add backup**. The **Contact information** page opens.

Reviewer 2: ```suggestion
7. If you did not select a hosting backup, a pop-up may display asking you to confirm your choice. Click **Continue without backup** or **Add backup**. The **Contact information** page opens.
```

8. Select the domain owner from your existing contacts, or choose to create a new contact.
    <Lightbox image={image2} alt="" />

9. Scroll down to the bottom of the page and click **Confirm**.

>     If you chose to create a new contact, you will be required to enter their details. Do so, and click **Create Handle**. Once completed, the **Summary** page opens.

Reviewer: ```suggestion
    If you choose to create a new contact, enter their details, then click **Create Handle**. The **Summary** page opens.
```
